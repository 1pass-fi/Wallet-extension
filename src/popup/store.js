import { composeWithDevTools } from 'redux-devtools-extension'
import { createStore, applyMiddleware } from 'redux'
import thunk from 'redux-thunk'

import rootReducer from './reducers/index'

const store = createStore(
  rootReducer,
  composeWithDevTools(applyMiddleware(thunk))
)

export default store
