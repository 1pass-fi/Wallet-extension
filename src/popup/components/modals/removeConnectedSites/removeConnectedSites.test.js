import '@babel/polyfill'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import React from 'react'
import { Provider } from 'react-redux'
import store from 'popup/store'

import RemoveConnectedSites from '.'

describe('Test for RemoveConnectedSites component', () => {
  describe('Render without crashing', () => {
    it('renders correctly', () => {
      const sites = [
        'koi.rocks',
        'app.uniswap.org'
      ]

      const { container } = render(<Provider store={store}><RemoveConnectedSites sites={sites} accountName={'Account 1'} /></Provider>)
      expect(container).toMatchSnapshot()
    })
  })
})
