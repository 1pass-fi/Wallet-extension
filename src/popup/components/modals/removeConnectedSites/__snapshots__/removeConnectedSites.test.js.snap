// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for RemoveConnectedSites component Render without crashing renders correctly 1`] = `
<div>
  <div
    class="modal-container"
  >
    <div
      class="modal undefined"
    >
      <div
        class="modal-close-icon"
      >
        <svg />
      </div>
      <div
        class="modal-title remove-sites-modal"
      >
        Account 1
         is connected to these sites.
        <br />
        They can view your account address.
      </div>
      <div
        class="connected-site-row"
        style="background: rgb(238, 238, 238);"
      >
        <div
          class="connected-site-name"
        >
          koi.rocks
        </div>
        <button
          class="delete-button"
        >
          <div
            class="delete-icon"
          >
            <svg />
          </div>
        </button>
      </div>
      <div
        class="connected-site-row"
        style="background: rgb(255, 255, 255);"
      >
        <div
          class="connected-site-name"
        >
          app.uniswap.org
        </div>
        <button
          class="delete-button"
        >
          <div
            class="delete-icon"
          >
            <svg />
          </div>
        </button>
      </div>
      <div
        class="manual-connect-container"
        style="background: rgb(238, 238, 238);"
      >
        <button
          class="manual-connect-button"
        >
          <div
            class="plus-icon"
          >
            <svg />
          </div>
        </button>
        <div
          class="manual-connect-label"
        >
          manually connect to current site
        </div>
      </div>
    </div>
  </div>
</div>
`;
